### ISSUE TITLE
[Bug]: ZL and ZR only report "1" for a single frame before reverting back to "0"

### ISSUE THREAD

**Issue by HTV04** [lovebrew/LovePotion#156]
### Software Version

2.2.0 (Latest)

### What Happened?

ZL and ZR only report "1" for a single frame before reverting back to "0", even if they're held down.

### What was Expected?

ZL and ZR should report "1" while they're held down.

### Code to Reproduce

```Lua
-- If held down, returns "1" for a single frame, then returns "0" after
love.joystick.getJoysticks()[1]:getGamepadAxis("triggerleft")
```

### Console

Nintendo Switch

### Firmware Version(s)

12.0.1

### Custom Firmware Version

0.19.3

### Execution Method

Homebrew Menu

### Code of Conduct

- [X] I have read the Code of Conduct and agree to these terms

**Comment by TurtleP** [comment]
This should be fixed in 3de41e0 -- please check with the GitHub Actions artifact if you can.

**Comment by HTV04** [comment]
It works, thanks!
